src/Component/Tests/Dummy/DummyResourceWithSections.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\Tests\Dummy;
+
+use Sylius\Component\Resource\Metadata\Create;
+use Sylius\Component\Resource\Metadata\Index;
+use Sylius\Component\Resource\Metadata\Resource;
+use Sylius\Component\Resource\Metadata\Show;
+
+#[Resource(alias: 'app.dummy', section: 'admin')]
+#[Index]
+#[Create]
+#[Resource(alias: 'app.dummy', section: 'shop')]
+#[Index]
+#[Show]
```

Comment:
I'm still thinking how should we handle this. Even tho, I'm on a boat with the idea of having `#[Index]` as separate property, I'm still not convinced on resolution of them just by its placement under the resource. Can we discuss this design decision

Reply:
Yeah good idea, we have to find a solution that we are both proud first.

Reply:
Do you have a proposal?